css-animations-2/Overview.bs

```diff
@@ -690,7 +693,173 @@ The 'animation' shorthand property syntax is as follows:
 
 <span class=prod><dfn>&lt;single-animation></dfn> = <<'animation-duration'>> || <<easing-function>> || <<'animation-delay'>> || <<single-animation-iteration-count>> || <<single-animation-direction>> || <<single-animation-fill-mode>> || <<single-animation-play-state>> || [ none | <<keyframes-name>> ] || <<single-animation-timeline>></span>
 
+<h2 id="animation-triggers">
+Declaring Animation Triggers</h2>
 
+    Animation Triggers are defined using the 'animation-trigger-*' properties.
+    These list-valued properties,
+    which are all [=longhands=] of the 'animation-trigger' [=shorthand=],
+    form a [=coordinating list property group=]
+    with 'animation-name' as the [=coordinating list base property=]
+    and each item in the [=coordinated value list=]
+    defining the properties of a single animation trigger.
+
+## The 'animation-trigger-type' property ## {#animation-trigger-type}
+
+The 'animation-trigger-type' property specifies the [=animation trigger type=]
+of the [=animation trigger=].
+
+<pre class='propdef'>
+Name: animation-trigger-type
+Value: <<single-animation-trigger-type>>#
+Initial: once
+Applies to: all elements
+Inherited: no
+Percentages: N/A
+Computed value: list, each item a keyword as specified
+Animation type: not animatable
+Canonical order: per grammar
+</pre>
+
+<span class=prod><dfn>&lt;single-animation-trigger-type></dfn> = once | repeat | alternate | state</span>
+
+## The 'animation-trigger-timeline' property ## {#animation-trigger-timeline}
+
+The 'animation-trigger-timeline' property specifies the <a>timeline</a>
+of the animation’s associated [=animation trigger=].
+
+<pre class='propdef'>
+Name: animation-trigger-timeline
+Value: <<single-animation-timeline>>#
+Initial: auto
+Applies to: all elements
+Inherited: no
+Percentages: N/A
+Computed value: list, each item either
+  the keyword ''single-animation-timeline/none'',
+  the keyword ''single-animation-timeline/auto'',
+  a case-sensitive [=css identifier=],
+  a computed ''scroll()'' function,
+  or
+  a computed ''view()'' function
+Canonical order: per grammar
+Animation Type: not animatable
+</pre>
+
+Issue: Probably a trigger with timeline "none" is under-specified here. Need to clarify what it means.
```

Comment:
Well, I remember we considered this in a previous issue specifically on syncing animations, but we said there that this will be the stuff that will eventually be done via GroupEffects. That said, if you see some way of changing the definition here and keeping this more open for later extensions I'm all for it.

Reply:
> but unlike animation delay could use absolute times since page load or other events I suppose?

I feel like we'll still the pseudo-selector you suggested in https://github.com/w3c/csswg-drafts/issues/8175, for example, to defer until DOM and assets are loaded. This is pretty common.

Reply:
OK, but `none` here is part of the `single-animation-timeline` syntax. Should we remove it just here? Or entirely?

Reply:
I see.. I suppose suppose we can keep it and I guess the animation would never start?

Reply:
Yeah, I guess it resets state back to `idle`.